technical-reports/Performance-Testing.md

```diff
@@ -0,0 +1,72 @@
+# Continuous Profiling in TEAMMATES
```

Comment:
- The header does not match the file name. Change either one.
- You can drop the "Continuous"